src/style-spec/reference/v8.json

```diff
@@ -6061,6 +6070,82 @@
       },
       "property-type": "data-constant"
     },
+    "raster-color": {
+      "type": "color",
+      "doc": "Defines a color map by which to colorize a raster layer, parameterized by the `[\"raster-value\"]` expression and evaluated at 1024 uniformly spaced steps over the range specified by `raster-color-range`.",
+      "transition": false,
+      "sdk-support": {
+        "basic functionality": {
+          "js": "2.12.0"
+        },
+        "data-driven styling": {}
+      },
+      "expression": {
+        "interpolated": true,
+        "parameters": [
+          "raster-value"
+        ]
+      },
+      "property-type": "color-ramp"
+    },
+    "raster-color-mix": {
+      "type": "array",
+      "default": [
+        0.2126,
```

Comment:
How did you choose these default values?

Reply:
Ah okay I see it in the PR description. Maybe it could be mentioned in the documentation section of this property?

Reply:
Ah yes, for RGB imagery input, luminosity seemed, visually at least, like the least surprising starting point. And note that for grayscale input (e.g. b&w [clouds](https://studio.mapbox.com/tilesets/rreusser.dv2uv1dw/#0.86/0/0) or [radar](https://studio.mapbox.com/tilesets/rreusser.5vzfcdjn/#0.86/0/-97.5)), the components add up to one so that [1, 0, 0, 0] = [0, 1, 0, 0] = [0, 0, 1, 0] = [1/3, 1/3, 1/3, 0] = [0.2126, 0.7152, 0.0722, 0] are all equivalent.

Good point. I'll add a note 👍